packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,211 @@
+
+### Logo tips
+
+* If your logo isn't in one of the acceptable formats, ask your design team to convert your existing logo to the preferred PNG format. If you have difficulty with converting your image to an acceptable format, contact the [OIN team](mailto:oin@okta.com).
+
+* If your logo isn't square, consider using your website favicon. Alternatively, you can use the first letter of your app wordmark and convert it to a square image.
+
+* You can use a slightly non-square icon, but take into account the scaling that occurs when your logo is fitted into a 200 x 200 pixels space. If you have additional logo questions, contact the [OIN team](mailto:oin@okta.com).
+
+<div class="full">
+
+![OIN acceptable logos](/img/oin/oin-logo-size-guide.png)
+
+</div>
+
+### Examples of acceptable logos
+
+<div class="full">
+
+![OIN acceptable logos](/img/oin/oin-logo-guide.png)
+
+</div>
+
+## App description guidelines
+
+The content in the OIN Manager **General Settings** > **App description** field appears in your Integration detail page (under the **Overview** heading) when your integration is published in the OIN catalog. See the [OIN catalog](https://www.okta.com/integrations/) &nbsp;for examples of existing descriptions in their associated Integration detail page.
+
+In the **App description** field, describe the value that you provide to customers by partnering with Okta. Your description needs to be less than 1024 characters. See a description example in this [ServiceNow Integration detail](https://www.okta.com/integrations/servicenow-ud/) &nbsp;page.
+
+## Use case guidelines
+
+Integrations are organized by use cases in the OIN catalog. From the OIN Manager, you can specify the use case categories that your integration supports in the **General Settings** > **App use case** field. You can select up to five use cases. Use the following description list to determine the appropriate use case category for your integration:
+
+| Use case | Integration capability |
+| -------- | ---------------------- |
+| [Single Sign-On](https://www.okta.com/integrations/?category=single-sign-on) (most common) | Enables users to access your application from any device with a single entry of their Okta user credentials. This use case is appropriate for SAML and OpenID Connect (OIDC) integrations. |
+| [Automation](https://www.okta.com/integrations/?category=automation)  | Automates business processes and Okta administration tasks. Most integrations in this use case are API service integrations that access Okta’s APIs using OAuth 2.0.  |
+| [Centralized Logging](https://www.okta.com/integrations/?category=centralized-logging) | Aggregates Okta logs into a central location, like a Security Information and Event Management (SIEM) tool, for optimized searching and alerting capabilities. Integrations in this use case are API service integrations that poll the Okta API for System Logs using OAuth 2.0. |
```

Comment:
I'm not sure if it would be confusing to people from whom English isn't their first language, but what if:

"Integrations that fall under this use case are API ..."
otherwise maybe:
"API service integrations that poll the Okta API for System Logs using OAuth 2.0 support this use case." (or similar)

Reply:
I'll use the latter suggestion. Thanks!